scripts/survey/README.md

```diff
@@ -0,0 +1,26 @@
+## Migrations
+
+#### Steps to run the script files
+
+This script is intended as a remedy to push completed surveySubmission records to kafka and it is related to the 6.0.0 branch
+
+### Step 1:
+
+    Navigate to /opt/projects/scripts/survey/
+
+### Step 2:
+
+Run the script to delete duplicate projects.
```

Comment:
@borkarsaish65 this is wrong. please update

Reply:
changed